packages/expo-splash-screen-command/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog
```

Comment:
I'm not sure whether we want changelog in this package 🤔 It's actually a part of `expo-splash-screen` so any changes should go there as well 🤔